llvm/test/CodeGen/AMDGPU/llc-pipeline.ll

```diff
@@ -1005,6 +1016,7 @@
 ; GCN-O2-NEXT:        SI lower SGPR spill instructions
 ; GCN-O2-NEXT:        Virtual Register Map
 ; GCN-O2-NEXT:        Live Register Matrix
+; GCN-O2-NEXT:        Machine Register Class Info Analysis
```

Comment:
Still a lot of avoidable recomputations happening, need to addPreserved in more places

Reply:
Go through all the passes and add `AU.addPreserved<RegisterClassInfoWrapperPass>()` in getanalyisfsage